app/dao/email_branding_dao.py

```diff
@@ -3,6 +3,17 @@
 from app.models import EmailBranding
 
 
+def dao_get_existing_alternate_email_branding_for_name(name):
+    """
+    Returns any email branding with name of format `{name} (alternate {x})`
+
+    where name is the provided name and x is an integer starting at 1
+    """
+    return (
+        EmailBranding.query.filter(EmailBranding.name.ilike(f"{name} (alternate %)")).order_by(EmailBranding.name).all()
```

Comment:
nit: I feel like the implication here is the ordering should be based numerically on x but it'll actually be lexicographic/alphabetical.

Reply:
Good point - in this case the ordering doesn't actually matter for our use case since we immediately chuck all the names in a set anyway, but that's definitely something to be aware of.